vortex-ffi/src/lib.rs

```diff
@@ -18,6 +18,10 @@ pub use log::vx_log_level;
 use tokio::runtime::{Builder, Runtime};
 use vortex::error::VortexExpect;
 
+#[cfg(feature = "mimalloc")]
+#[global_allocator]
+static GLOBAL: mimalloc::MiMalloc = mimalloc::MiMalloc;
```

Comment:
The allocator is configured once per binary so it's seems unsure that ffi should be configuring it

Reply:
We should look into this further. But I think that c-ffi linked packages use a different heap and therefore different allocator.